### ISSUE TITLE
About the data decompression

### ISSUE THREAD

**Issue by lixin4ever** [DavidGrangier/wikipedia-biography-dataset#1]
Hello,
        I try to decompress the .z files follow the instructions in README but the system always raises the error
`unzip:  cannot find zipfile directory in one of tmp.zip or
        tmp.zip.zip, and cannot find tmp.zip.ZIP, period.`
Have you ever encountered this problem?

**Comment by BibhuDas123** [comment]
Yes, i am also facing same problem.The data can't be exracted for the compressed files..

**Comment by prathik-naidu** [comment]
Also getting the same problem, @lixin4ever @BibhuDas123 did you get it working?

**Comment by wetneb** [comment]
These instructions are designed for UNIX (linux, mac) systems. They work for me.

**Comment by wetneb** [comment]
@prathik-naidu @BibhuDas123 @lixin4ever If you use Windows, have you tried with `type` instead of `cat`?

**Comment by Nicoleqwerty** [comment]
You must download the whole files and zip them together...I think...

**Comment by wetneb** [comment]
@Nicoleqwerty no the files are already compressed - you must concatenate them together, as explained in the instructions. If you need help let me know what your operating system is.

**Comment by tanooj-s** [comment]
You have to concatenate all the files, so the commands you have to run are explicitly:
`cat wikipedia-biography-dataset.z?? > tmp.zip`
`unzip tmp.zip`

**Comment by ha-lins** [comment]
downloading the whole files still cannot work.how do you solve it? @Nicoleqwerty

**Comment by dinghaij** [comment]
in the unix/linux os , following the instructions of the README.md, i.e.
cat wikipedia-biography-dataset.z?? > tmp.zip
unzip tmp.zip
rm tmp.zip
i have tried this and it does works.
